<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSliderClass</class>
 <widget class="QWidget" name="ParameterSliderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>55</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ParameterSlider</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QSlider" name="slider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3">
    <widget class="QCheckBox" name="autoBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="3">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources/main.qrc">
       <normaloff>:/new/prefix1/refresh.png</normaloff>:/new/prefix1/refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="minLabel">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="maxLabel">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParameterSliderClass</receiver>
   <slot>textChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParameterSliderClass</receiver>
   <slot>sliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderChanged(int)</slot>
  <slot>textChanged(int)</slot>
 </slots>
</ui>
